.gitlab/ci/build_quartz.yml

```diff
@@ -67,16 +67,19 @@ clang_9_0_0 (Master build_with_deps_on_quartz):
   variables:
     COMPILER: "clang@9.0.0"
     SPEC: "@develop%${COMPILER}"
+    MIRROR: "/usr/workspace/smithdev/mirror"
```

Comment:
This variable should be declared globally, avoiding repetition.
See 'variables' section in  `.gitlab-ci.yml`.

Reply:
Thanks for the tip! I'm slowly learning how all of this fits together...

Reply:
That’s what reviews are for :)